Localisation/Localiser.cs

```diff
@@ -114,14 +114,16 @@ public static void SetDefaultCulture(string defaultCultureName)
 
 		public static void SwitchLanguage(string cultureName)
 		{
-			CurrentLocale = new CultureInfo(cultureName);
-			currentLocalisationTable = LoadLocalisationTable(cultureName);
-
-			if (currentLocalisationTable == null)
+			var newTable = LoadLocalisationTable(cultureName);
```

Comment:
newTable sounds like its being created but everything suggests its not new but just loaded